docs/changelog.md

```diff
@@ -1,6 +1,25 @@
 Change Log
 ==========
 
+## Version 1.8.0
+
+_2021-03-22_
+
+ * New: Restructure interop artifacts, starting with `kotlinx-metadata`.
```

Comment:
@ZacSweers want to expand on this a little more? Please feel free to push into the same branch.